javascript/Task4/task4.js

```diff
@@ -0,0 +1,27 @@
+let arr = [];
+let maxSeconds = null;
+
+let random = Math.round((Math.random() * (9 - 1 + 1) + 1));
```

Comment:
(9 - 1 + 1) what is it?

Reply:
again, we can move this randomizer to separate function and reuse it in the function below.